src/components/SelectInput/ReactSelect.scss

```diff
@@ -0,0 +1,72 @@
+@import '../../styles/variables/forms.scss';
```

Comment:
This import shouldn't be necessary since variables are applied to the :root. Commenting out this import has no impact on the component.